Author: Alastair McKinstry <mckinstry@debian.org>
Description: Mark XFAIL a known bug with bottleneck < 1.3.0
 See: https://github.com/pydata/xarray/issues/5342
Last-Updated: 2021-08-25
Forwarded: not-needed

--- a/xarray/tests/test_duck_array_ops.py
+++ b/xarray/tests/test_duck_array_ops.py
@@ -873,8 +873,10 @@
     np.testing.assert_allclose(residuals, [2.0])
 
 
+# XFAIL fixed in bottleneck 1.3.0
 @requires_dask
 @requires_bottleneck
+@pytest.mark.xfail
 def test_push_dask():
     import bottleneck
     import dask.array
